.github/workflows/plugin_upstream_testing_base.yml

```diff
@@ -25,6 +25,8 @@ jobs:
     steps:
       - name: "Check out repository code"
         uses: "actions/checkout@v2"
+        with:
+          ref: "refs/remotes/origin/${{ matrix.nautobot-version == 'next' && 'next-2.0' || 'develop' }}"
```

Comment:
Can we switch `develop` to `$GITHUB_REF_NAME` just on the off chance the repo's default is not `develop`

Reply:
`env.GITHUB_REF_NAME` is empty in this workflow context when testing from pull request ihttps://github.com/nautobot/nautobot-plugin-secrets-providers/pull/78

Some info:
https://stackoverflow.com/questions/70901255/using-env-variable-github-ref-name-doesnt-give-me-branch-name

I added hardcoded `develop` as a fallback, it works this way.